doc/common_info.md

```diff
@@ -506,7 +506,7 @@ bazelrc_lines=()
 
 if [[ $OSTYPE == darwin* ]]; then
   xcode_path=$(xcode-select -p)
-  xcode_version=$(xcodebuild -version | tail -1 | cut -d " " -f3)
+  xcode_version=$(/usr/bin/xcodebuild -version 2>/dev/null | head -1 | cut -d " " -f2)
   xcode_build_number=$(/usr/bin/xcodebuild -version 2>/dev/null | tail -1 | cut -d " " -f3)
```

Comment:
@BalestraPatrick

Reply:
Ah good catch regarding the lack of use 👍

Reply:
Sorry for the delay, but I looked at this now and noticed the following:
- `xcode_version` with the above change will be equal to `15.1`. This isn't great because if you're using a beta version of Xcode (and possibly have multiple versions installed) it won't be deterministic. For someone it could pick beta 1 and for someone else it could pick beta 2.
- `xcode_build_number` instead gives the unique build number for an Xcode version (which changes between betas of the same version as well), such as `15C65`. 

I think in this script we likely want to replace the usage of `$xcode_version` with `$xcode_build_number` instead (and of course remove `xcode_version=(...)` since it'll then be unused). I see that `rules_xcodeproj`  (cc: @brentleyjones) also does something similar (where `XCODE_PRODUCT_BUILD_VERSION` looks like `15C65`): https://github.com/MobileNativeFoundation/rules_xcodeproj/blob/9a6ec001d7ac96bb8474d95b0ecab909db9dab06/xcodeproj/internal/templates/bazel_build.sh#L86-L95